modules/graphstudio/pages/map-data-to-graph.adoc

```diff
@@ -42,40 +41,34 @@ This section contains subsections for each data source.
 * AWS S3 - <<_create_s3_data_source>>
 * GCS - <<_create_a_gcs_data_source>>
 
-=== Add local data file
-
+[tabs]
+====
+Local File::
++
+--
 After clicking btn:[Local File], you will be prompted to choose one or more local files to upload through the GUI.
```

Comment:
Can we talk about the supported files here?
- CSV file
- TSV file
- JSON Lines file

Also talk about the unsupported file
- Zip files
- Tar files